environ/environ.go

```diff
@@ -108,34 +109,34 @@ func (e *Environ) load(s store.Store, service string, collisions *[]string, noPa
 
 // Load loads environment variables into e from s given a service
 // collisions will be populated with any keys that get overwritten
-func (e *Environ) Load(s store.Store, service string, collisions *[]string) error {
-	return e.load(s, service, collisions, false)
+func (e *Environ) Load(ctx context.Context, s store.Store, service string, collisions *[]string) error {
```

Comment:
as it stands, anywhere we add in `ctx` here as a starting argument is considered a breaking change since it alters the public interface.

I know we're preparing the default branch as a candidate for v3, so maybe its moot, but does it make sense to call this out heavily somewhere (either in a changelog, or updating the merge commit) to designate the nature of these breaking changes?

Reply:
I'm fine updating the commit to be a breaking change in conventional commit-ese. It is certainly a significant break for anyone using chamber as a library, which I wasn't thinking about.

Reply:
Commit updated to call out that this is a breaking change.

Reply:
As for `<Action>WithContext`: I'm OK with using the breaking change opportunity to avoid adding those extra functions to preserve compatibility. It'd certainly be a lot of extra functions.